docker-compose.yml

```diff
@@ -9,6 +14,12 @@ services:
       - ${PORT:-3000}:3000
     depends_on:
       - postgres
+      - api-dummy
+
+  api-dummy:
```

Comment:
this is inherited into `docker-compose.ci.yml` so it could be removed from there?
(because `docker-compose.ci.yml` is always used in conjunction with `docker-compose.yml`, never alone)

Reply:
Not sure what 'this' and 'it' are?

So we don't need as much in `docker-compose.ci.yml` (i.e. take the entries out of here that occur in `docker-compose.yml`) ?

Reply:
When using `docker-compose -f docker-compose.yml -f docker-compose.ci.yml ...`, the `api-dummy` service is present due to this definition, so as you say you can take the entries out of `docker-compose.ci.yml` that occur in `docker-compose.yml`.

Reply:
These have now been tidied. We had to put `fakes3` back into the `docker-compose.yml` becaue this is needed when developing for `start-services`